easycv/models/detection/yolox/yolox.py

```diff
@@ -165,3 +166,28 @@ def forward_compression(self, x):
         outputs = self.head(fpn_outs)
 
         return outputs
+
+
+@MODELS.register_module
+class YOLOXExport(YOLOX):
```

Comment:
using yoloXExport to fit the jit trace input/output requirenments, otherwise we should change the original YOLOX and add a config for it to open the post process.